docs/content-server-api/deployments/get.md

```diff
@@ -0,0 +1,385 @@
+# Get all deployments
+
+List all deployments stored in the content server
+
+**URL** : `/deployments`
+
+<!-- **URL Parameters** : `pk=[integer]` where `pk` is the ID of the Account on the server. -->
+
+**Method** : `GET`
+
+**Auth required** : NO
+
+**Permissions required** : NO
+
+**Query Parameters** :
+
+- fromLocalTimestamp
+  - Format: int
+  - Value: timestamp
+  - Example: fromLocalTimestamp=1606829553969
+  - Default value: NULL - no lower filter is set
+  - Description: Acts as a filter in the collection of deployments, this value is the minimum value of local timestamp that any deployment in the collection will have.
+
+- toLocalTimestamp
+  - Format: int
+  - Value: timestamp
+  - Example: toLocalTimestamp=1606829553969
+  - Default value: NULL - no upper filter is set
+  - Description: Acts as a filter in the collection of deployments, this value is the maximum value of local timestamp that any deployment in the collection will have.
+
+- entityType
+  - Format: string array
+  - Value: [scene, profile, wearable]
+  - Example: entityType=scene&entityType=profile
+  - Default value: scene, profile & wearable
+  - Description: The type of entities that will be shown in the collection, many values can be sent. If any string ends with an ‘s’ or has whitespaces, then it will be correctly parsed. If any of the entity types sent is invalid, then the request will return a 404 status code.
+
+- entityId
+  - Format: string array
+  - Value: ContentFileHash
+  - Example: entityId=QmNknKv8MuKbfZ73z4QdUEsNbTd1ZAN1fSuwTFGiNGeCt5
+  - Default value:
+  - Description: It is the hash value of the entire deployment object.
+
+- onlyCurrentlyPointed
+  - Format: boolean
+  - Value: true/false
+  - Default value:
+  - Description: Enable this filter to remove from the view the deployments that have been deleted by a new one.
+
+- deployedBy
+  - Format: string array
+  - Value: ethAddress
+  - Example: deployedBy=0xdbd3b0f6a8a9eb2bb5d968d26e5565a4448919ac
+  - Default value:
+  - Description: The Address associated with the deployment. This filter is used to obtain deployments from the same user for example.
+
+- sortingField
+  - Format: string
+  - Value: [local_timestamp, entity_timestamp]
+  - Example: sortingField=entity_timestamp
+  - Default value: local_timestamp
+  - Description: This value is used as the field to order all the deployments in the collection. If no parameter is sent, then the default field to order with will be local_timestamp.
+
+- sortingOrder
+  - Format: string
+  - Value: [ASC,DESC]
+  - Example: sortingOrder=ASC
+  - Default value: DESC
+  - Description: This value is used as the field to order all the deployments in the collection. If no parameter is sent, then the default order will be desc.
+
+- showAudit
```

Comment:
Let's hide this. It was necessary for a migration, but we should even remove it from the code